src/components/ListArea/ListArea.js

```diff
@@ -0,0 +1,35 @@
+import './ListArea.css';
+import React, {useContext, useEffect, useState} from 'react';
+import {ListContext} from '../../context'
+import _ from 'lodash'
+
+function ListArea({title, listIndex=-1, readonly=false}){
+    
+    const [value, setValue] = useState()
+    const {listA, listB, setListByIndex, getListByIndex} = useContext(ListContext)
```

Comment:
you can a hook function to your context to make this prettier, and export less code
so instead of ```const {stuff} = useContext(StuffContext);``` you'll get ```const {stuff} = useStuffContext();```